### PR TITLE
Add Packaged Grain to the Getting Grain page

### PR DESCRIPTION
**Pull request by phated**
Probably needs some cleanup by @ospencer

### CODE REVIEW

File: src/getting_grain.md [hunk 1]
```diff
@@ -2,16 +2,56 @@
...
+
+### Windows x64 - Download
+
+You can [download it](https://github.com/grain-lang/grain/releases/download/grain-v0.3.0/grain-win-x64.exe) directly from GitHub.
+
+You'll either want to put it into your path or keep it inside your project and invoke with `./grain-win-x64.exe`.
+
+## Building Grain from Source
+
+To get access to the entirely native compiler, you can build Grain from source.
```
Comment (ospencer) [inline src/getting_grain.md:47]: This implies that packaged isn't native, but we didn't say that explicitly. Should we say something about it? I could go either way.

**Review by ospencer — COMMENTED** [review]
Honestly LGTM! I see this page evolving in the future 🤩 

Left you one note that I'm curious your thoughts on, but otherwise I think this is good to ship!

File: src/getting_grain.md (hunk 1 above)
Comment (phated) [reply src/getting_grain.md:51]: Yeah, I think we could mention that. Let me see if I can come up with a good way to phrase it.

File: src/getting_grain.md [hunk 2]
```diff
@@ -2,16 +2,60 @@
...
-## Building Grain
+We have a couple different ways to get Grain. Most developers will prefer the Packaged Grain binaries, but make sure to read the note about speed!
 
-Grain currently only works on Mac and Linux. We are hard at work trying to integrate Binaryen and OCaml on Windows, but this is uncharted territory! Please let us know if you have this low-level expertise.
+## Packaged Grain
 
-To build the compiler, you'll need [Node.js](https://nodejs.org/en/download/current/) v14 and [Yarn](https://yarnpkg.com/getting-started/install).
+The Grain toolchain (including our CLI, compiler, runtime, and standard library) is shipped as a single binary. Binaries are available for [MacOS x64](#MacOS-x64---Homebrew), [Linux x64](#Linux-x64---Download), and [Windows x64](#Windows-x64---Download).
+
+__Note:__ These binaries are a bit slow the first time you build a project (around 70 seconds) because they build and output the runtime & standard library into your project. If you need extra speed, you can build the native compiler from source! See [Building Grain from Source](#Building-Grain-from-Source) below.
```
Comment (phated) [inline src/getting_grain.md:11]:
```suggestion
__Note:__ These binaries are a bit slow the first time you build a project (around 70 seconds). The reason for this is that the compiler is actually running in JavaScript, _and_ it is building and writing the runtime & standard library into your project. If you need extra speed, you can build the native compiler from source! See [Building Grain from Source](#Building-Grain-from-Source) below.
```

@ospencer What do you think about this mentioning it is a JS compiler and then linking to the Building from Source section which mentions native compiler.

File: src/getting_grain.md (hunk 1 above)
Comment (phated) [reply src/getting_grain.md:51]: See above?

File: src/getting_grain.md (hunk 2 above)
Comment (ospencer) [reply src/getting_grain.md:11]: We decided this on Discord:
```suggestion
__Note:__ These binaries are a bit slow when first building a project (around 70 seconds). The packaged compiler is running in JavaScript, _and_ it builds and writes the runtime & standard library into your project. If you need raw speed, you can build the native compiler from source! See [Building Grain from Source](#Building-Grain-from-Source) below.
```

**Review by ospencer — APPROVED** [review]

**Pull request merged**